README.md

```diff
@@ -326,11 +326,11 @@ en_marche_mailer:
         # can use same old sound connexions settings and will be prepend
     mail_aggregator:
         routing_keys:
-            - 'em_mails_*' # default
+            - 'em_mails.*.*' # default
             # or
-            #- 'em_mails_campaign_*
-            #- 'em_mails_transactional_*
-            #- 'em_mails_transactional_app_name
+            #- 'em_mails.campaign.*
+            #- 'em_mails.transactional.*
+            #- 'em_mails.transactional.app_name
```

Comment:
should it be `em_mails.transactional.app_name.*` ?

Reply:
nop, just 3 levels, prefix, mail type and app_name
https://github.com/EnMarche/mailer-bundle/blob/6025591f94b42f6dd704a33eb1ee868f7832d476/src/Mailer/Transporter/AmqpMailTransporter.php#L53-L60